storybook/main.ts

```diff
@@ -20,16 +20,23 @@ const getStoryPathsFromEnv = (): string[] | false => {
   }
   return [storyPath]
 }
-
 const defaultStoryPaths = [
   "../(docs|draft-packages|packages)/**/*.mdx",
   "../(docs|draft-packages|packages)/**/*.stories.tsx",
 ]
-
 const config = {
   stories: getStoryPathsFromEnv() || defaultStoryPaths,
-  addons: ["@storybook/addon-essentials", "@storybook/addon-a11y"],
-  staticDirs: [{ from: "./assets", to: "/static/media" }],
+  addons: [
+    "@storybook/addon-essentials",
+    "@storybook/addon-a11y",
+    "@storybook/addon-mdx-gfm",
```

Comment:
Pretty sure we don't need this either

Reply:
I assume you specifically mean the `@storybook/addong-mdx-gfm`? I think it was added during the update flow, but I can remove it again (I hope w/o problems ;) )

Reply:
Yeah we don't need this plugin added by the migration :)